exercises/ruby-1-a/ruby_1_a_example.rb

```diff
@@ -1 +1,11 @@
-TODO
\ No newline at end of file
+VOWELS = %w{a e i o u}
+def word_with_most_vowels(sentence)
```

Comment:
Have you tried to see if there are multiple valid approaches?

Reply:
Nope. I just blindly copied what you've done and changed it to Ruby :)

This won't be the actual Ruby exercise so I'm not worrying too much. I imagine there would be though. I would definitely not use a single method for example.